src/templates/view-edit-toolbar.html

```
@@ -155,190 +155,201 @@ <h5 class='modal-title' id='procurementRequestModalLabel'>{{ 'Create procurement
+    {% endif %}
+
+    <div class='vertical-separator'></div>
 
-    {# Calendar booking #}
-    {% if Entity.entityType.value == 'items' and (Entity.entityData.is_bookable or not Entity.isReadOnly) %}
-      {% if Entity.entityData.is_bookable %}
-        <a class='btn mr-2 btn-transparent' title='{{ 'Book item'|trans }}' aria-label='{{ 'Book item'|trans }}' href='scheduler.php?items[]={{ Entity.id }}'>
-          <i class='fas fa-calendar-plus fa-fw'></i>
+    {% if mode == 'edit' %}
+      {# VIEW button #}
+        <a title='{{ 'Save'|trans }}' data-action='update-entity-body' aria-label='{{ 'Save'|trans }}' class='btn btn-primary mr-2'>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)